### PR TITLE
Kia: add FW versions for KIA_K5_2021

### PR DESCRIPTION
**Pull request by sky77764**
I added fingerprint info for KIA_K5_2021.

### CODE REVIEW

**Comment by sunnyhaibin** [comment]
Make sure to provide the dongle ID from your device.

**Comment by sky77764** [comment]
> Make sure to provide the dongle ID from your device.

Dongle ID: d1e6ea79a8f86e9d

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -1377,28 +1377,37 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
...
+      b'\xf1\x00DL3 MFC  AT KOR LHD 1.00 1.04 99210-L2000 210527',
     ],
     (Ecu.abs, 0x7D1, None): [
       b'\xf1\000DL ESC \006 101 \004\002 58910-L3200',
       b'\xf1\x8758910-L3200\xf1\000DL ESC \006 101 \004\002 58910-L3200',
       b'\xf1\x8758910-L3800\xf1\x00DL ESC \t 101 \x07\x02 58910-L3800',
       b'\xf1\x8758910-L3600\xf1\x00DL ESC \x03 100 \x08\x02 58910-L3600',
       b'\xf1\x00DL ESC \t 100 \x06\x02 58910-L3800',
+      b'\xf1\x00DL ESC \x01 104 \x07\x12 58910-L2200',
+      b'\xf1\x8758910-L2200\xf1\x00DL ESC \x01 104 \x07\x12 58910-L2200',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1402]:
```suggestion
```

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -1377,28 +1377,37 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
...
       b'\xf1\x8799110L2100\xf1\x00DL3_ SCC FHCUP      1.00 1.03 99110-L2100         ',
       b'\xf1\x00DL3_ SCC F-CUP      1.00 1.03 99110-L2100         ',
+      b'\xf1\x00DL3_ SCC FHCUP      1.00 1.03 99110-L2100         ',
     ],
     (Ecu.eps, 0x7D4, None): [
       b'\xf1\x8756310-L3110\xf1\000DL3 MDPS C 1.00 1.01 56310-L3110 4DLAC101',
       b'\xf1\x8756310-L3220\xf1\x00DL3 MDPS C 1.00 1.01 56310-L3220 4DLAC101',
       b'\xf1\x8757700-L3000\xf1\x00DL3 MDPS R 1.00 1.02 57700-L3000 4DLAP102',
       b'\xf1\x00DL3 MDPS C 1.00 1.01 56310-L3220 4DLAC101',
+      b'\xf1\x8756310-L2220\xf1\x00DL3 MDPS C 1.00 1.02 56310-L2220 4DLDC102',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1387]:
```suggestion
```

File: selfdrive/car/hyundai/values.py [hunk 3]
```diff
@@ -1407,6 +1416,11 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
       b'\xf1\x87SCMSAA8572454GK1\x87x\x87\x88Vf\x86hgwvwvwwgvwwgT?\xfb\xff\x97fo\xffH\xb8\xf1\x81U913\x00\x00\x00\x00\x00\x00\xf1\x00bcsh8p54  U913\x00\x00\x00\x00\x00\x00TDL4T16NB05\x94t\x18',
       b'\xf1\x87954A02N300\x00\x00\x00\x00\x00\xf1\x81T02730A1  \xf1\x00T02601BL  T02730A1  WDL3T25XXX730NS2b\x1f\xb8%',
       b'\xf1\x00bcsh8p54  U913\x00\x00\x00\x00\x00\x00TDL4T16NB05\x94t\x18',
+      b'\xf1\x00HT6TA261BLHT6TAB00A1SDL0C20KS0\x00\x00\x00\x00\x00\x00\\\x9f\xa5\x15',
+      b'\xf1\x87RBNSLH969121U13Zwx\x87\x88\x8a\xaa\xaa\xaa\x86fhfvwwgvf\x7f\xff\x9a\xff_\xf7VK\xf1\x81HT6TAB00A1',
+      b'SDL0C20KS0\x00\x00\x00\x00\x00\x00',
+      b'\xf1\x87RBNSLH969121U13Zwx\x87\x88\x8a\xaa\xaa\xaa\x86fhfvwwgvf\x7f\xff\x9a\xff_\xf7VK\xf1\x89HT6TAB00A1',
+      b'\xf1\x82SDL0C20KS0\x00\x00\x00\x00\x00\x00',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1423]:
```suggestion
```

File: selfdrive/car/hyundai/values.py [hunk 4]
```diff
@@ -1377,28 +1377,37 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
...
+      b'\xf1\x00DL ESC \x01 104 \x07\x12 58910-L2200',
+      b'\xf1\x8758910-L2200\xf1\x00DL ESC \x01 104 \x07\x12 58910-L2200',
     ],
     (Ecu.engine, 0x7E0, None): [
       b'\xf1\x87391212MKT0',
       b'\xf1\x87391212MKV0',
       b'\xf1\x870\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf1\x82DLDWN5TMDCXXXJ1B',
+      b'\xf1\x81HM6M2_0a0_DQ0',
+      b'DLBUD0CMB3XWSDQB',
+      b'\xf1\x82DLBUD0CMB3XWSDQB',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1410]:
```suggestion
```

**Comment by sshane** [comment]
Interesting, can't find this part number online anywhere except eBay, so can't compare to figure out how it's different yet. LGTM

**Pull request merged**